uint8 FOOTSTEP_PLANNER_SUCCESS  =  0
uint8 FOOTSTEP_PLANNER_ACTIVE   =  1
uint8 FOOTSTEP_PLANNER_FAILED   =  2
uint8 FOOTSTEP_PLANNER_ERROR    =  4
uint8 FOOTSTEP_INVALID_START    =  8
uint8 FOOTSTEP_INVALID_GOAL     = 12
uint8 FOOTSTEP_VALID_GOAL       = 16
uint8 FOOTSTEP_INVALID_STEP     = 32
uint8 FOOTSTEP_MISMATCHED_TIMES = 64

uint8 FOOTSTEP_EXECUTION_SUCCESS  =    0
uint8 FOOTSTEP_EXECUTION_FEEDBACK =  128
uint8 FOOTSTEP_EXECUTION_ACTIVE   =  129
uint8 FOOTSTEP_EXECUTION_FAILED   =  130

uint8 status
string status_msg
